docker-compose.yml

```diff
@@ -25,3 +25,21 @@ services:
     environment:
       - K6_OUT=influxdb=http://influxdb:8086/k6
     command: 'version'
+
+  deb:
```

Comment:
Hm, maybe we should have these in a separate docker compose file? Otherwise they'll be run every time someone uses `docker-compose.yml` to run InfluxDB+Grafana, no?

Reply:
True. Missed that.